app/controllers/members_controller.rb

```diff
@@ -1,14 +1,20 @@
 class MembersController < ApplicationController
-  before_action :authenticate_user!, only: [:make_admin, :remove_admin, :destroy]
+  before_action :authenticate_user!, except: [:index]
   before_action :set_group
-  before_action :set_group_user, except: [:index]
+  before_action :set_group_user, except: [:index, :create]
   before_action :set_is_admin
   before_action :check_admin_rights, except: [:index]
```

Comment:
Ah, my mistake -- for some reason I thought this was the `GroupsController` when I wrote that comment. Please ignore :)